src/consensus/params.h

```diff
@@ -29,6 +29,11 @@ struct BIP9Deployment {
     int64_t nStartTime;
     /** Timeout/expiry MedianTime for the deployment attempt. */
     int64_t nTimeout;
+    /** Delay lock in occuring until at least this timestamp.
```

Comment:
Should this say median time not a block timestamp (`nTime`) ?

IIUC correctly the problem mentions [here](https://github.com/bitcoin/bitcoin/pull/21377#issuecomment-792425221) is solved for the following rational. If a reorg occurs across a retarget period with new block timestamps rolling back the median time, the LOCKED_IN period serves a safety buffer until ACTIVATION ? If so that would be great to document it somewhere.

Reply:
No longer relevant, but fwiw I was using "timestamp" in the sense of "epoch timestamp, ie seconds since 1970", not "block timestamp, ie nTime value".